megamek/src/megamek/client/ui/swing/dialog/imageChooser/AbstractIconChooser.java

```diff
@@ -85,8 +85,6 @@ private void initialize(JTree tree) {
         splitPane = new JSplitPane(JSplitPane.HORIZONTAL_SPLIT, true, scrpTree, scrpImages);
         splitPane.setResizeWeight(0.5);
 
-        splitPane.setDividerLocation(GUIPreferences.getInstance().getInt(GUIPreferences.IMAGE_CHOOSER_SPLIT_POS));
```

Comment:
?

Reply:
Missed the MMPreferences side of the swapover, nice catch.